### ISSUE TITLE
Support multiple custom queries

### ISSUE THREAD

**Issue by shikhar** [confluentinc/kafka-connect-jdbc#113]
Currently while you can have the connector configured to ingest from multiple tables, only a single custom query may be specified (https://github.com/confluentinc/kafka-connect-jdbc/blob/ccbfc8e2d3a835e742b9bd465b662d2779a11c76/src/main/java/io/confluent/connect/jdbc/JdbcSourceConnector.java#L118-L139).

A use-case where this is somewhat limiting: to parallelize ingest for a large table by having `WHERE` clauses on non-overlapping ranges, but then this means having to run multiple connector instances given this restriction.

For having `query` accept a list of multiple queries (or maybe we should have a new `queries` field),`;` should work well as a delimiter.

**NOTE** `topic.prefix` has special semantics when `query` is specified, doc: "Prefix to prepend to table names to generate the name of the Kafka topic to publish data to, or in the case of a custom query, the full name of the topic to publish to." -- so even in the case of multiple queries I think we'd want that the data is all destined to the same topic.

**Comment by malonej7** [comment]
I agree the use of multiple queries per connector would be useful, however our use case is a bit different.  We need to create 1 connector per table due to the fact that we need to utilize DB functions and can't just read in every column as-is.  Having a ton of connectors running has been causing issues on our system as well as becoming unmanageable.  Therefore it'd be nice to be able to use one connector for multiple queries, but each query would need its own name and would have to be routed to its own topic.  

The config for this could get pretty ugly so I can see it being tough to implement.  In the meantime we are going to try to utilize database views instead of queries and build off of the table.whitelist functionality.
